cylc/flow/network/schema.py

```diff
@@ -1735,7 +1735,9 @@ class Arguments:
             description='Hold all tasks after the specified cycle point.',
             required=True
         )
-
+        flow_num = Int(
+            description='Number of flow to hold.'
+        )
```

Comment:
Should the default be `flow=all`?

The pattern used in other mutations is a field called `flow` of type `Flow` which accepts either an integer or a string. See `FlowMutationArguments` in this file.

In this case `new` and `none` don't make sense, so it might need a new type.